README.md

```diff
@@ -188,7 +188,7 @@ To view a translated version
 - Copy the templated index file so it's ready for translation:
 
   ```shell
-  cp public/index-embed-v1.html build/index.html
+  mkdir -p build && cp public/index-embed-v1.html build/index.html
```

Comment:
New developers may not run any of the build scripts before, so this ensures the build folder exists before file copying.